internal/services/containerapps/helpers/container_apps.go

```diff
@@ -2240,6 +2257,20 @@ func SecretsDataSourceSchema() *pluginsdk.Schema {
 		Sensitive: true,
 		Elem: &pluginsdk.Resource{
 			Schema: map[string]*pluginsdk.Schema{
+				"identity": {
+					Type:        pluginsdk.TypeString,
+					Computed:    true,
+					Sensitive:   false,
```

Comment:
This doesn't need to be sensitive

Reply:
this wasn't marked as sensitive? but I have gone through and updated the sensitivity for the properties so that only `value` is sensitive. for some reason `name` was marked as sensitive previously